apis/extension/numa_aware.go

```diff
@@ -171,6 +169,12 @@ type PodCPUAlloc struct {
 
 type PodCPUAllocs []PodCPUAlloc
 
+type CPUSharedPool struct {
```

Comment:
This share pool is not referenced?

Reply:
koordlet only needs to report pools to help diagnose, and koordlet will bind cpus per NUMANode of NUMANodeResources. This is also the current koordlet implementation.